.all-contributorsrc

```diff
@@ -49,7 +49,7 @@
     },
     {
       "login": "jmclean-cnexus",
-      "name": "jmclean-cnexus",
+      "name": "Jon McLean",
```

Comment:
Will you change your display name in the README too? Thanks!

Reply:
sure...i didnt touch it bc of the "Do not remove or modify this section", haha.

Reply:
Yeah, that warning is just because everything in the section gets overwritten whenever All Contributors bot adds a new contributor.

So it would be updated with the correct name next time I add a contributor, but that might be in a while

Reply:
updated!